etc/LATISS/primary_hdu.header

```diff
@@ -91,24 +91,9 @@ IMAGETAG=                      / DAQ Image id
 OBSANNOT=                      / Observing annotation
 
         ---- Information from Camera ----
-CCD_MANU=                      / CCD Manufacturer
 CCD_TYPE=                      / CCD Model Number
```

Comment:
Is this supposed to be duplicated as its in the senor HDU file?

Reply:
That was a duplicate and has been removed.